templates/details.html

```diff
@@ -0,0 +1,114 @@
+{% extends "index.html" %}
```

Comment:
Create a `layout.html` file as you had in the tutorial which will have the common content for all the pages and both index and details will extend that file.